projects/pgai/tests/semantic_catalog/conftest.py

```diff
@@ -22,6 +22,17 @@ def load_postgres_air(container: PostgresContainer):
     ):
         cur.execute(script1)  # pyright: ignore [reportArgumentType]
         cur.execute(script2)  # pyright: ignore [reportArgumentType]
+    script3 = Path(__file__).parent.joinpath("data", "cagg.sql").read_text()
+    with (
+        psycopg.connect(
+            container.connection_string(database="postgres_air"),
+            autocommit=True,
+        ) as con,
+        con.cursor() as cur,
+    ):
+        for query in script3.split(";"):
+            if query.strip():
+                cur.execute(query)  # pyright: ignore [reportArgumentType]
```

Comment:
I was getting errors about cannot create materialized view within a transaction errors when I tried:
1. Sticking the execute in the above with block
2. Executing all the queries in one `cur.execute`.

Hence, I stuck this into its own context block (with `autocommit=True`) and had to do the `split(";")` so that it all worked.

Reply:
no problem